aten/src/ATen/cuda/tunable/GemmHipblaslt.h

```diff
@@ -142,6 +163,86 @@ int GetStrideCFromParams(const GemmStridedBatchedParams<T>* params) {
   return params->stride_c;
 }
 
+template <typename T, typename ParamsT>
+float GetAlphaFromParams(const ParamsT* params) {
+  return params->alpha;
```

Comment:
My answer here will be similar to the other questions.  The GemmHipblaslt.h header defines a templated implementation of hipblaslt gemm.  Rather than copy/paste a large chunk of code and tailor it for scaled_mm, the gemm and bgemm implementation came first and here I am adapting it for scaled gemm utilizing the same template.

The code looks like I'm doing partial function template specialization but it's really function overloading of different template functions with the same name.  T is used indirectly here because T matches the T used by the template parameter ParamsT which would be either GemmParams<T> or GemmStridedBatchedParams<T>.  The function with the same name that only has the single T template parameter is the overload specific for ScaledGemmParams<T>.  I was trying to save some copy/paste code, but perhaps it would be more clear if I split out the GemmParams<T> and GemmStridedBathcedParams<T> as their own overloads.

I will rewrite and break these out as 3 obvious overloads so that it is less confusing.